.github/workflows/build_book.yml

```diff
@@ -21,7 +21,7 @@ jobs:
         uses: styfle/cancel-workflow-action@0.9.1
         with:
           access_token: ${{ github.token }}
-      - uses: actions/checkout@v2
+      - uses: actions/checkout@v3
 
       # Install dependencies
       - name: Set up Python
```

Comment:
You can avoid the warnings by updating to `uses: actions/setup-python@v4`

Reply:
To clarify,  line 28.

Reply:
Thanks for checking this!